x-pack/plugins/uptime/public/components/functional/charts/duration_chart.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  AreaSeries,
+  Axis,
+  Chart,
+  CurveType,
+  getAxisId,
+  getSpecId,
+  LineSeries,
+  Position,
+  ScaleType,
+  timeFormatter,
+} from '@elastic/charts';
+import { EuiPanel, EuiTitle } from '@elastic/eui';
+import React, { useContext } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  convertMicrosecondsToMilliseconds as microsToMillis,
+  getChartDateLabel,
+} from '../../../lib/helper';
+import {
+  MonitorDurationAreaPoint,
+  MonitorDurationAveragePoint,
+} from '../../../../common/graphql/types';
+import { UptimeSettingsContext } from '../../../contexts';
+import { getColorsMap } from './get_colors_map';
+
+interface DurationChartProps {
+  /**
+   * Timeseries data that is used to express a max/min area series
+   * on the duration chart.
+   */
+  durationArea: MonitorDurationAreaPoint[];
+  /**
+   * Timeseries data that is used to express an average line series
+   * on the duration chart.
+   */
+  durationLine: MonitorDurationAveragePoint[];
+  /**
+   * The color to be used for the average duration series.
+   */
+  meanColor: string;
+  /**
+   * The color to be used for the range duration series.
+   */
+  rangeColor: string;
+}
+
+/**
+ * This chart is intended to visualize monitor duration performance over time to
+ * the users in a helpful way. Its x-axis is based on a timeseries, the y-axis is in
+ * milliseconds.
+ * @param props The props required for this component to render properly
+ */
+export const DurationChart = ({
+  durationArea,
+  durationLine,
+  meanColor,
+  rangeColor,
+}: DurationChartProps) => {
+  const { absoluteStartDate, absoluteEndDate } = useContext(UptimeSettingsContext);
+  // this id is used for the area chart representing the max/min of check durations
+  const areaSpecId = getSpecId('area');
+
+  // this id is used for the line chart representing the average duration length
+  const averageSpecId = getSpecId('average');
+
+  return (
+    <React.Fragment>
+      <EuiTitle size="xs">
+        <h4>
+          <FormattedMessage
+            id="xpack.uptime.monitorCharts.monitorDuration.titleLabel"
+            defaultMessage="Monitor duration"
+            description="The 'ms' is an abbreviation for milliseconds."
+          />
+        </h4>
+      </EuiTitle>
+      <EuiPanel>
+        <Chart className={'story-chart'}>
```

Comment:
So the rendering of the chart is by default `canvas` the `renderer` prop can be omitted as in this case on all the other chart configuration if you want

Reply:
Great point, I've removed all of the `className` and `renderer` calls.

Resolved by https://github.com/elastic/kibana/pull/34313/commits/46632691d1864e72fa8bfa195937be78ea5921a7.